pyproject.toml

```diff
@@ -11,14 +17,16 @@ before-all = [
 ]
 test-command = [
     "cd {project}/bindings/python",
-    "tox -e",
+    "python -m pip install -r test-requirements.txt",
+    "python -X dev -m pytest --log-level=DEBUG --log-cli-level=DEBUG",
+    "python -m mypy --config-file mypy-tox.ini tests",
 ]
 
 [tool.cibuildwheel.macos.environment]
 BOOST_BUILD_PATH = "/tmp/boost/tools/build"
 BOOST_ROOT = "/tmp/boost"
 BOOST_VERSION = "1.76.0"
-MACOSX_DEPLOYMENT_TARGET = "10.9"  # required for full C++11 support
+MACOSX_DEPLOYMENT_TARGET = "10.14"  # required for full C++17 support
```

Comment:
in `RC_2_0` we need C++14 support too...

Reply:
[10.9 is sufficient for C++14](https://cibuildwheel.readthedocs.io/en/stable/cpp_standards/#macos-and-deployment-target-versions), but I guess the comment is out of sync with the requirement.